test/cases/image_tests.sh

```diff
@@ -59,14 +59,10 @@ get_test_cases () {
             SKIP_CASES=("${SKIP_OSTREE[@]}")
         fi
 
-        # skip container test if running on old osbuild version
-        # b/c manifests have been modified to include the
-        # "manifest-lists" and "skopeo-index" options which are
-        # not yet supported on 8.8 and 9.2 downstream
-        if nvrGreaterOrEqual "osbuild" "83"; then
+        if nvrGreaterOrEqual "osbuild-composer" "84"; then
```

Comment:
I guess this function exits the whole test as it returns 1?

Reply:
The root cause is that 

`grep ova-boot <<< "$ALL_CASES"`

will exit with `1` on aarch64 b/c the ova image doesn't exist there. And because we're running with `set -e` that will terminate everything. Updated & will reschedule the pipeline.